bridge/checkpoint/transport/transport.go

```diff
@@ -62,15 +66,36 @@ func (t *libp2pGossipTransport) Subscribe(handler func(interface{})) error {
 
 		// and call handler
 
-		// protoMessage, ok := obj.(*proto.SignedMessage)
-		// if !ok {
-		// 	t.logger.Warn("received unexpected typed message", "message", obj)
+		protoMsg, ok := obj.(*proto.CheckpointMessage)
+		if !ok {
+			t.logger.Warn("received unexpected typed message", "message", obj)
```

Comment:
Why is this a warning and not an error?

Reply:
[Changed to Error](https://github.com/0xPolygon/polygon-edge/pull/468/commits/d22b06d42c1bb9df130990e51b2785d206ce5c84)